python/batch-predictor/requirements.txt

```diff
@@ -1,8 +1,6 @@
 findspark
 pyspark==3.0.1
-mlflow>=1.2.0,<=1.23.0
+mlflow>=1.26.1,<=2.0.0
 cloudpickle==2.0.0
 pyarrow>=0.14.1,<=9.0.0
-protobuf>=3.0,<4.0.0
```

Comment:
Should we continue to pin protobuf version? Or is it expected that the new mlflow sdk is compatible with protobuf 4.X ?

Reply:
According to the release note for Mlflow 1.26.1, it is expected to be compatible with protobuf 4.X. Although, i suppose we can pin it to <5.0.0 .